lib/stm32/l0/rcc.c

```diff
@@ -408,5 +540,50 @@ void rcc_set_hpre(uint32_t hpre)
 	RCC_CFGR = reg | (hpre << RCC_CFGR_HPRE_SHIFT);
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Common clock setups.
+ *
+ * @note After a reset, the system uses MSI@2.1MHz.
+ */
+
+void rcc_clock_setup_in_hsi16_out_32mhz(void)
+{
```

Comment:
I actually don't care for these either (hence just one). IMO these belong to examples, but the lib needs more examples for the 'more exotic' ones like L0 ;) I'm also not sure that they actually work across the whole family (e.g. L0 in this case).

Will remove this one (and add to my own code).